Tests/SystemTests/ChannelModesTests.swift

```diff
@@ -159,6 +159,8 @@ extension ChannelModesTests: SubscriberDelegate {
     
     func subscriber(sender: AblyAssetTrackingSubscriber.Subscriber, didChangeAssetConnectionStatus status: ConnectionState) {}
     
+    func subscriber(sender: AblyAssetTrackingSubscriber.Subscriber, didUpdateResolution resolution: Resolution) {}
+    
```

Comment:
Since you made that method optional, can't we remove it? :wink:

Reply:
changed a288514997ea39ae67ca98329302cc66012e4595